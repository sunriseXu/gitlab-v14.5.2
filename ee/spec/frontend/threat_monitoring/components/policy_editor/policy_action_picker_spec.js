import { shallowMount } from '@vue/test-utils';
import PolicyActionPicker from 'ee/threat_monitoring/components/policy_editor/policy_action_picker.vue';

describe('PolicyActionPicker component', () => {
  let wrapper;

  const factory = ({ propsData } = {}) => {
    wrapper = shallowMount(PolicyActionPicker, {
      propsData: {
        ...propsData,
      },
    });
  };

  beforeEach(() => {
    factory();
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('renders policy action picker', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
