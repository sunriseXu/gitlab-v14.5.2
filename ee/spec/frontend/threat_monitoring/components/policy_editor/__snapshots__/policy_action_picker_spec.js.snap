// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PolicyActionPicker component renders policy action picker 1`] = `
<div
  class="gl-bg-gray-10 gl-border-solid gl-border-1 gl-border-gray-100 gl-rounded-base gl-px-5 gl-pt-5"
>
  <gl-form-stub
    inline=""
  >
    <gl-sprintf-stub
      message="%{labelStart}Then%{labelEnd} %{action} %{spanStart}the network traffic.%{spanEnd}"
    />
  </gl-form-stub>
</div>
`;
